<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FXMenu</class>
 <widget class="QWidget" name="FXMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #202020;
font-family: &quot;Cascadia Code&quot;;
border: 0px;</string>
  </property>
  <widget class="QGroupBox" name="monitor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>121</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="grooveOff">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>343</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/sliderGrooveOff.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="grooveOn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>343</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/icons/sliderGrooveOn.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>41</width>
      <height>403</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider { background: transparent; }

QSlider::groove:vertical {
	border: 1px solid #050505;
	border-radius: 5px;
	background-color: #000000;
	width: 11px;
	margin: 32px 32px;
}

QSlider::handle:vertical {
	image: url(:/icons/slider.png);
	height: -32px;
	margin: -32px -32px;
}</string>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Listen to your own voice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>MIC MONITOR VOLUME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>grooveOn</zorder>
   <zorder>label</zorder>
   <zorder>grooveOff</zorder>
   <zorder>slider</zorder>
  </widget>
  <widget class="QGroupBox" name="hotkeys">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>611</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border: 1px solid #282828;
	border-radius: 10px;
	background: #282828;
	outline: 0;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QListWidget" name="hotkeysList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	border: 1px solid #282828;
	border-radius: 10px;
	background: #282828;
	outline: 0;
}

QListWidget::item:hover {
	background-color: #008080;
	border-radius: 10px;
}

QListWidget::item:selected {
	background-color: #006060;
	border-radius: 10px;
}
QListWidget::item:selected:!active {
	background-color: #006060;
	border-radius: 10px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QToolButton" name="remove">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>201</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove selected Hotkey&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	border: 0px;
	border-right: 1px solid #000000;
	background: #404040;
	border-top-left-radius: 10px;
}

QToolButton:hover {
	background: #505050;
}

QToolButton:pressed {
	background: #484848;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="add">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>201</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a new Hotkey&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
	border: 0px;
	border-left: 1px solid #000000;
	background: #404040;
	border-bottom-right-radius: 10px;
}

QToolButton:hover {
	background: #505050;
}

QToolButton:pressed {
	background: #484848;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>611</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {
	border: 1px solid #282828;
	background: #282828;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}

QTabBar::tab {
	background: #303030;
	border: 1px solid #005050;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	min-width: 70px;
	padding: 3px 8px;
}

QTabBar {
	icon-size: 9px 9px;
}

QTabBar::tab:selected {
	border-bottom-color: #282828;
}

QTabBar::tab:hover {
	background: #404040;
}

QTabBar::tab:!selected {
	margin-top: 2px;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="reverb">
    <property name="styleSheet">
     <string notr="true">QWidget {
	background: #282828;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}</string>
    </property>
    <attribute name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/FXOff.png</normaloff>:/icons/FXOff.png</iconset>
    </attribute>
    <attribute name="title">
     <string>REVERB</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="autotune">
    <property name="styleSheet">
     <string notr="true">QWidget {
	background: #282828;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}</string>
    </property>
    <attribute name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/FXOff.png</normaloff>:/icons/FXOff.png</iconset>
    </attribute>
    <attribute name="title">
     <string>AUTOTUNE</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pitch">
    <property name="styleSheet">
     <string notr="true">QWidget {
	background: #282828;
	border-bottom-left-radius: 10px;
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}</string>
    </property>
    <attribute name="icon">
     <iconset resource="../appicon.qrc">
      <normaloff>:/icons/FXOff.png</normaloff>:/icons/FXOff.png</iconset>
    </attribute>
    <attribute name="title">
     <string>PITCH</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="overlay" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>284</y>
     <width>611</width>
     <height>187</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,100);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
border-top-right-radius: 10px;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../appicon.qrc"/>
 </resources>
 <connections/>
</ui>
